scripts/fuzz_bots.py

```diff
@@ -79,22 +80,35 @@ async def _async_runner(
     return out_result
 
 
-def run_fuzz_bots():
-    """Runs fuzz bots"""
+def run_fuzz_bots(argv: Sequence[str] | None = None) -> None:
+    """Runs fuzz bots
```

Comment:
period at the end

Reply:
nvm it's missing in a few places outside your changes; we can fix this with a global lint later